shacl12-core/index.html

```diff
@@ -5691,6 +5692,194 @@ <h4>sh:property</h4>
 					</p>
 				</section>
 				
+				<section id="MemberShapeConstraintComponent">
+					<h4>sh:memberShape</h4>
+					<p>
+						<code>sh:memberShape</code> requires that each <a>value node</a> be a <a href="#syntax-rule-SHACL-list">SHACL list</a>,
+						and that each member of that list conforms to the given <a>node shape</a>.
+					</p>
+					<p>
+						<span class="component-class">Constraint Component IRI</span>: <code>sh:MemberShapeConstraintComponent</code>
+					</p>
+
+					<div class="parameters">Parameters:</div>
+					<table class="term-table">
+						<tr>
+							<th>Property</th>
+							<th>Summary and Syntax Rules</th>
+						</tr>
+						<tr>
+							<td><code>sh:memberShape</code></td>
+							<td>
+								The shape that all members of the <a href="#syntax-rule-SHACL-list">SHACL list</a> must conform to.
+								<span data-syntax-rule="memberShape-node">The value of <code>sh:memberShape</code> must be a <a>well-formed</a> <a>node shape</a>.</span>
+							</td>
+						</tr>
+					</table>
+					<div class="def def-text">
+						<div class="def-header">TEXTUAL DEFINITION</div>
+						<div class="def-text-body" data-validator="MemberShape">
+							Let <code>$memberShape</code> be a <a>value</a> of <code>sh:memberShape</code>.
+							Each <a>value node</a> <code>v</code> must be a <a href="#syntax-rule-SHACL-list">SHACL list</a> - a <a>failure</a> MUST be produced if <code>v</code> is not a SHACL list.
+							For each member <code>m</code> of the list <code>v</code>
+							a <a>failure</a> MUST be produced if the <a>conformance checking</a> of <code>m</code> against <code>$memberShape</code> produces a <a>failure</a>.
+							There must be at most one <a>validation result</a> for each list <code>v</code>.
```

Comment:
Should this end something like "...for each list v and triggering constraint," (indicating cross-product) since `memberShape` could be used to link multiple shapes?